.travis.yml

```diff
@@ -1,36 +1,49 @@
-os: linux
+os:
+  - linux
+  - osx
+  - windows
 dist: trusty
-language: python
-python: 3.6
-
-env:
- - TRAVIS_NODE_VERSION=12
+language: shell 
+cache: pip
 
 services:
  - mongodb
 
-matrix:
+env: 
+  jobs:   
+   - MAKE_COMMAND=test
+   - MAKE_COMMAND=integration-test
+   - MAKE_COMMAND=package
+  global: 
+    - DEPLOY=true
+      TRAVIS_NODE_VERSION=12
+      WINDOWS_TRAVIS_PYTHON_VERSION=3.6.8
+
+
+jobs:
   include:
-   - os: linux
-     python: 3.6
-     env:
-      - DEPLOY=true
-      - TRAVIS_NODE_VERSION=12
-   - os: osx
-     osx_image: xcode10.1
-     language: generic
-     python: null
-     env:
-      - DEPLOY=true
-      - TRAVIS_NODE_VERSION=12
-   - os: windows
-     language: sh
-     python: 3.6.8
-     env:
-      - DEPLOY=false  # set to true when tested and AppVeyor is deprecated
-      - TRAVIS_NODE_VERSION=12
-      - TRAVIS_PYTHON_VERSION=3.6.8
+    - os: linux
+      language: python
+      python: 3.6
+      # Install nvm and use correct node version
+      before_install:
+        - sudo apt-get install libqt5xml5;
+        - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $TRAVIS_NODE_VERSION;
+    - os: osx
+      osx_image: xcode11.2 # Contains a modern (3.7.4) installed python version
```

Comment:
Unfortunately it also breaks the PyInstaller builds for older macOS versions. We need to use the oldest xcode/macOS version that we want to support.

Reply:
Okay that's great to know.